examples/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.4"
```

Comment:
I'm all for using the latest `docker-compose` syntax version, but maybe we could pin at `3` to increase availability/adoption.